### DISCUSSION TITLE
Is there a way to apply the Breath profile from Konsole to Kitty (in Manjaro Linux)?

### DISCUSSION THREAD

**Discussion by s-aleck** [kovidgoyal/kitty#4398] [Q&A] [answered]
Konsole just has a nice, dull color scheme with the Breath profile. I've been trying out various Kitty color schemes, but some stuff is still brighter and flashier in Kitty than in Konsole. Is there a way to apply the Breath profile from Konsole to Kitty (in Manjaro Linux)?

**Comment by page-down** [comment, marked as answer]
Yes, if you don't mind doing it yourself for the theme you like.

Download kitty theme "template.conf" from themes repository.

https://github.com/kovidgoyal/kitty-themes

Find the corresponding colors in the breath theme's source code, convert them to hexadecimal color codes, and fill in the kitty theme configuration file.

https://gitlab.manjaro.org/artwork/themes/breath

https://gitlab.manjaro.org/artwork/themes/breath/-/blob/master/colors/BreathDark.colors

https://gitlab.manjaro.org/artwork/themes/breath/-/blob/master/konsole/Breath.colorscheme

Load your breath theme file in the kitty.conf configuration file via "include".
